pandas-stubs/core/frame.pyi

```diff
@@ -326,23 +327,39 @@ class DataFrame(NDFrame, OpsMixin):
     @overload
     def to_html(
         self,
-        buf: FilePathOrBuffer | None,
-        columns: Sequence[_str] | None = ...,
-        col_space: int | list[int] | dict[_str | int, int] | None = ...,
+        buf: FilePath | WriteBuffer[str],
+        columns: Sequence[HashableT] | None = ...,
+        col_space: ColspaceArgType | None = ...,
         header: _bool = ...,
         index: _bool = ...,
         na_rep: _str = ...,
-        formatters=...,
-        float_format=...,
+        formatters: list[Callable[[object], str]]
+        | tuple[Callable[[object], str], ...]
+        | Mapping[Hashable, Callable[[object], str]]
+        | None = ...,
+        float_format: Callable[[float], str] | None = ...,
         sparsify: _bool | None = ...,
         index_names: _bool = ...,
-        justify: _str | None = ...,
+        justify: Literal[
+            "left",
+            "right",
+            "center",
+            "justify",
+            "justify-all",
+            "start",
+            "end",
+            "inherit",
+            "match-parent",
+            "initial",
+            "unset",
+        ]
+        | None = ...,
         max_rows: int | None = ...,
         max_cols: int | None = ...,
         show_dimensions: _bool = ...,
         decimal: _str = ...,
         bold_rows: _bool = ...,
-        classes: _str | list | tuple | None = ...,
+        classes: _str | Sequence[str] | None = ...,
```

Comment:
`Sequence[str]` will match `_str` so the `_str` is redundant.

Reply:
Fixed.